packages/salesforcedx-utils-vscode/src/services/sourceTrackingService.ts

```diff
@@ -45,4 +50,156 @@ export class SourceTrackingService {
+    const statusResults = this.statusOutputRows.map(row =>
+      this.resultConverter(row)
+    );
+
+    if (statusResults.length === 0) {
+      return nls.localize('no_local_or_remote_changes_found');
+    }
+
+    const sortedStatusResults = statusResults.sort(this.rowSortFunction);
+
+    return new StatusResultsTable(sortedStatusResults).value();
+  }
+
+  /**
+   * STL provides a more useful json output.
+   * This function makes it consistent with the Status command's json.
+   */
+  private resultConverter = (input: StatusOutputRow): StatusResult => {
+    const { fullName, type, ignored, filePath, conflict } = input;
+    const origin = SourceStatusSummary.originMap.get(input.origin) || 'Local';
+    const actualState = SourceStatusSummary.stateMap.get(input.state);
+    return {
+      fullName,
+      type,
+      // this string became the place to store information.
+      // The JSON now breaks out that info but preserves this property for backward compatibility
+      state: `${origin} ${actualState}${conflict ? ' (Conflict)' : ''}`,
+      ignored,
+      filePath,
+      origin,
+      actualState,
+      conflict
+    };
+  };
+
+  private static originMap = new Map<
+    StatusOutputRow['origin'],
+    StatusResult['origin']
+  >([
+    ['local', 'Local'],
+    ['remote', 'Remote']
+  ]);
+
+  private static stateMap = new Map<
+    StatusOutputRow['state'],
+    StatusResult['actualState']
+  >([
+    ['delete', 'Deleted'],
+    ['add', 'Add'],
+    ['modify', 'Changed'],
+    ['nondelete', 'Changed']
+  ]);
+
+  // sort order is state, type, fullname
+  private rowSortFunction = (a: StatusResult, b: StatusResult): number => {
+    if (a.state.toLowerCase() === b.state.toLowerCase()) {
+      if (a.type.toLowerCase() === b.type.toLowerCase()) {
+        return a.fullName.toLowerCase() < b.fullName.toLowerCase() ? -1 : 1;
+      }
+      return a.type.toLowerCase() < b.type.toLowerCase() ? -1 : 1;
+    }
+    return a.state.toLowerCase() < b.state.toLowerCase() ? -1 : 1;
+  };
+}
+
+class StatusResultsTable {
+  private static baseColumns = [
+    { label: 'STATE', key: 'state' },
+    { label: 'FULL NAME', key: 'fullName' },
+    { label: 'TYPE', key: 'type' },
+    { label: 'PROJECT PATH', key: 'filePath' }
+  ];
+
+  private columns = this.statusResults.some(result => result.ignored)
+    ? [{ label: 'IGNORED', key: 'ignored' }, ...StatusResultsTable.baseColumns]
+    : StatusResultsTable.baseColumns;
+
+  constructor(private statusResults: StatusResult[]) {}
+
+  public value(): string {
+    this.statusResults.forEach(statusResult =>
+      this.convertToTableRow(statusResult)
+    );
+
+    const table: string = new Table().createTable(
+      (this.statusResults as unknown) as Row[],
+      this.columns
+    );
+
+    return table;
+  }
+
+  private convertToTableRow(result: StatusResult): FormattedStatusResult {
+    return Object.assign(result, {
+      ignored: result.ignored ? 'true' : 'false',
```

Comment:
Updated this based on Randi's findings from QA.  At this point in time, now matches up with the cli output from force:source:status:
<img width="250" alt="2023-05-01_22-17-43" src="https://user-images.githubusercontent.com/46458081/235585021-97228894-24e9-49d5-a3a4-bce2fa8c00e3.png">
<img width="250" alt="2023-05-01_22-17-30" src="https://user-images.githubusercontent.com/46458081/235585046-beaaedbd-3383-4c0b-9460-e392d8d6cf9e.png">